### DISCUSSION TITLE
Can I customize the blocking page LOGO

### DISCUSSION THREAD

**Discussion by epaphrasmakoko** [openappsec/openappsec#229] [Feature Requests]
How can i change the logo of open appsec in the blocking page

**Comment by orianelou** [comment]
Hi, 

This is currently an open feature request, keep this open to collect interest, I will update here once it will be added. 

Thank you for the great sugesstion!

**Comment by orianelou** [comment]
If you're interested in this feature, please 👍 or upvote this issue to show your support!

**Comment by shenazy** [comment]
yes, we really need that feature

**Comment by maroshaj02** [comment]
I think it is a good idea coz the logo should reflect the functionality of the tool,

**Comment by claudpizzaro** [comment]
This would be a great feature for use homelab enthusiast for protection out critial homelab services and it will hide which WAF we're using behind
